workspaces/src/network/server.rs

```diff
@@ -67,3 +70,17 @@ impl Drop for SandboxServer {
             .unwrap();
     }
 }
+
+/// Turn off neard-sandbox logs by default. Users can turn them back on with
+/// NEAR_ENABLE_SANDBOX_LOG=on and specify further paramters with the custom
+/// NEAR_SANDBOX_LOG for higher levels of specificity. NEAR_SANDBOX_LOG args
+/// will be forward into RUST_LOG environment variable as to not conflict
+/// with similar named log targets.
+fn supress_sandbox_logs_if_required() {
+    match std::env::var("NEAR_ENABLE_SANDBOX_LOG") {
+        Ok(val) if val == "on" => {}
```

Comment:
why not just require the env variable to be set? is the `on` pattern documented somewhere?

Reply:
I agree, the usual convention is either set/unset or 1/0

Reply:
Ahh, will switch to just set/unset. Way too use to `cmake` flags so a force of habit